Design

indydedeken opened the pull request:
Fix some issues from #31 : 
:ok: Position of my search have its own marker 
:ok: background on hover is now a little of opacity
:ok: back icon is now an svg file

indydedeken committed:
Fix the header bar on top

again…….. boring

indydedeken committed:
Fix multiple bugs about starred functionality

indydedeken committed:
CSS - starred view

indydedeken committed:
Merge branch 'design' of https://github.com/Phyks/VelibFxos into design

indydedeken committed:
test sur starred_stations

indydedeken committed:
Fix : back icon + index:hover

fix some #31

indydedeken committed:
[Search] Default icon for my position

Position of my search have its own marker

indydedeken committed:
back@2x is useless

because it’s svg

Phyks commented:
I see the marker for my position in the search view, but not for the search position. Else, I'm ok to merge it, thanks :)

indydedeken commented:
@Phyks Aw ok, I misunderstood you

indydedeken committed:
icon for searched position

- icon are a little customized for bikes view, stands view.
- add an icon on the searched position
#31

indydedeken commented:
@Phyks ok with the behaviour ?
